openvpn/factory.go

```diff
@@ -34,16 +41,9 @@ func NewServerConfig(
 	return &config
 }
 
-func NewClientConfig(
-	remote string,
-	caCertPath, tlsCryptKeyPath string,
-	protocol string,
-) *ClientConfig {
-	config := ClientConfig{NewConfig()}
-	config.SetClientMode(remote, 1194)
-	config.SetProtocol(protocol)
-	config.SetTLSCACertificate(caCertPath)
-	config.SetTLSCrypt(tlsCryptKeyPath)
+func newClientConfig(configDir string) *ClientConfig {
+	config := ClientConfig{NewConfig(configDir)}
+
 	config.RestrictReconnects()
```

Comment:
Strange that 2 values are here, and others in `NewClientConfigFromSession `

Reply:
This function is bigger - the diff only shows two lines